### PR TITLE
Cucumber Fever

### PR DESCRIPTION
**Pull request by AnywayFarus**
## About The Pull Request

Not without the help of my friend spriter, I added cucumbers, their seeds, the cultivation itself, so that they could be pickled and washed with a brine jar. I also added a Danish hot dog because it required cucumbers (perhaps that was the end goal), changed a couple of recipes to include cucumbers or pickles. Cucumbers have been added to both cargo orders and bounty cubes, as well as for the food order console

I think the Cucumber Update deserves its plush toy.

gg18b4b2cab0

## Why It's Good For The Game

I think more food and drink... would add quite a nice role playing experience, and additional gameplay for hydroponics.

## Changelog

:cl: Vishenka0704 and Ying-The-Pando
add: Cucumbers and pickles
add: Danish hot dog
balance: add cucumbers in dishes where they need
qol: add to bounty cubes, orders - new vegetables
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
You bastard, you had me thinking you were actually adding pickle rick.

**Comment by WhateverDimeIs** [comment]
make it scare felinids

**Comment by MidoriWroth** [comment]
Looks great! Was wondering when someone would add cucumbers to my Greek salad 😁 
I’m on mobile, but cucumbers in cargo/bounties/botany is typo’d

File: code/modules/cargo/bounties/botany.dm [hunk 1]
```diff
@@ -221,3 +221,7 @@
 		return FALSE
 	var/obj/structure/bonfire/B = O
 	return !!B.burning
+
+/datum/bounty/item/botany/cucumber
+	name = "Cucambers"
```
Comment (Jolly-66) [inline code/modules/cargo/bounties/botany.dm:226]:
```suggestion
	name = "Cucumbers"
```

**Review by Jolly-66 — COMMENTED** [review]
Just to help Midori here,

**Comment by Profakos** [comment]
Finally, the greek salad is complete...

**Comment by tralezab** [comment]
### This has to go
image

**Comment by AnywayFarus** [comment]
да это жестко

**Comment by optimumtact** [comment]
HE TURNED HIMSELF INTO A CUCUMBER JASON? FUNNIEST SHIT I HAVE EVER SEEN!

**Comment by optimumtact** [comment]
> ### This has to go
> 
> image

it must STAY

File: code/game/machinery/computer/chef_orders/order_datum.dm [hunk 2]
```diff
@@ -120,6 +120,19 @@
...
 
+/datum/orderable_item/cucumbers
+	name = "Cucumber"
+	category_index = CATEGORY_FRUITS_VEGGIES
+	item_instance = /obj/item/food/grown/cucumber
+	cost_per_order = 10
+
+
+/datum/orderable_item/pickles
+	name = "Pickles Jar"
```
Comment (Fikou) [inline code/game/machinery/computer/chef_orders/order_datum.dm:131]: it'd be "jar of pickles" or "pickle jar", not "pickles jar"

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_salad.dm [hunk 3]
```diff
@@ -130,6 +130,7 @@
 		/obj/item/food/onion_slice/red = 2,
 		/obj/item/food/cheese/wedge = 1,
 		/datum/reagent/consumable/quality_oil = 5,
+		/obj/item/food/grown/cucumber = 1
```
Comment (Fikou) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_salad.dm:133]: keep a trailing comma here

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_seafood.dm [hunk 4]
```diff
@@ -100,7 +100,8 @@
 		/obj/item/food/meat/cutlet = 3,
 		/obj/item/food/grown/cabbage = 1,
 		/obj/item/food/tofu = 1,
-		/datum/reagent/consumable/soysauce = 1
+		/datum/reagent/consumable/soysauce = 1,
+		/obj/item/food/grown/cucumber = 1
```
Comment (Fikou) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_seafood.dm:104]: and here

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_seafood.dm [hunk 5]
```diff
@@ -113,7 +114,8 @@
 		/obj/item/food/fishmeat = 1,
 		/obj/item/food/grown/cabbage = 1,
 		/obj/item/food/tofu = 1,
-		/datum/reagent/consumable/soysauce = 1
+		/datum/reagent/consumable/soysauce = 1,
+		/obj/item/food/grown/cucumber = 1
```
Comment (Fikou) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_seafood.dm:118]: and here

**Comment by AnywayFarus** [comment]
@optimumtact
solve it among yourselves, but give me the result, I can return it.

**Comment by Iamgoofball** [comment]
@AnywayFarus optimumtact is the headcoder

**Comment by bluetshirtguy** [comment]
ehhhhhhhh, it's just another plant for a few small food items, a bad joke and nothing else to it. I don't think adding plants with zero chemical depth or no gimmicks that could interact with other mechanics in neat ways is a good idea.

**Comment by Farquaar** [comment]
> ehhhhhhhh, it's just another plant for a few small food items, a bad joke and nothing else to it. I don't think adding plants with zero chemical depth or no gimmicks that could interact with other mechanics in neat ways is a good idea.

This attitude is the reason why cooking/hydroponics on Baystation is so boring and featureless.

**Comment by optimumtact** [comment]
A CUCUMBER JASON

**Pull request merged**